src/components/posts/Facebook/socialMedia/AdDisplay.js

```diff
@@ -20,5 +23,33 @@ export default function AdDisplay(props) {
         </p>
       </div>
     );
-  return <div id="shown-ad">Will display the {selectedAd} ad</div>;
+  return (
+    <div id="shown-ad">
+      <img
+        src={adData[selectedAd].img}
+        alt={`${selectedAd} clipart`}
+        className="ad-image"
+      ></img>
+      <h2>{adData[selectedAd].header}</h2>
```

Comment:
It's convention to only have one of the h1 - h6 on the page at a time, and `Navbar.js`'s subtitle already is an h2. Do you want to make this an h3 instead?